lib/logger/lib/logger/formatter.ex

```diff
@@ -134,6 +134,7 @@ defmodule Logger.Formatter do
   The color of the message can also be configured per message via
   the `:ansi_color` metadata.
   """
+  @spec new(keyword) :: {Logger.Formatter, struct()}
```

Comment:
I am not sure if we are not exposing implementation details here. Maybe `{module(), struct()}` but that's hardly helpful, I would skip this one.

Reply:
Yeah, that works for me!